index.d.ts

```diff
@@ -1,4 +1,4 @@
-import axios from 'axios'
+import * as axios from 'axios'
```

Comment:
I might be wrong but this might not necessary since we are using `esModuleInterop`

Reply:
If you revert this change and run `tsc --strict index.d.ts` the compiler will complain about `axios` not being a namespace. This is also why I added the typechecking to CI. `esModuleInterop` only changes how code is imported at runtime.